content/help/ops/setup/validation/index.md

```diff
@@ -0,0 +1,320 @@
+
+`x509: certificate signed by unknown authority` related errors are
+typically caused by an empty `caBundle` in the webhook
+configuration. Verify that it is not empty (see [verify webhook
+configuration](#invalid-configuration-is-accepted)). The
+`istio-galley` deployment consciously reconciles webhook configuration
+used the `istio-galley-configuration` `configmap` and root certificate
+mounted from `istio.istio-galley-service-account` secret in the
+`istio-system` namespace.
+
+1. Verify the `istio-galley` pods(s) are running:
+
+    {{< text bash >}}
+    $  kubectl -n istio-system get pod -listio=galley
+    NAME                            READY     STATUS    RESTARTS   AGE
+    istio-galley-5dbbbdb746-d676g   1/1       Running   0          2d
+    {{< /text >}}
+
+1. Verify you’re using Istio version >= 1.0.0. Older version of galley
+   did not properly re-patch the `caBundle`. This typically happened
+   when the `istio.yaml` was re-applied, overwriting a previously
+   patched `caBundle`.
+
+    {{< text bash >}}
+    $ for pod in $(kubectl -n istio-system get pod -listio=galley -o jsonpath='{.items[*].metadata.name}'); do \
+        kubectl -n istio-system exec ${pod} -it /usr/local/bin/galley version| grep ^Version; \
+    done
+    Version: 1.0.0
+    {{< /text >}}
+
+    1. Check the galley pod logs for errors. Failing to patch the `caBundle`
+       should print an error.
+
+    {{< text bash >}}
+    $ for pod in $(kubectl -n istio-system get pod -listio=galley -o jsonpath='{.items[*].metadata.name}'); do \
+        kubectl -n istio-system logs ${pod} \
+    done
+    {{< /text >}}
+
+1. If the patching failed, verify the RBAC configuration for galley.
+
+    {{< text bash yaml >}}
+    $ kubectl get clusterrole istio-galley-istio-system -o yaml
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      labels:
+        app: istio-galley
+      name: istio-galley-istio-system
+    rules:
+    - apiGroups:
+      - admissionregistration.k8s.io
+      resources:
+      - validatingwebhookconfigurations
+      verbs:
+      - '*'
+    - apiGroups:
+      - config.istio.io
+      resources:
+      - '*'
+      verbs:
+      - get
+      - list
+      - watch
+    - apiGroups:
+      - '*'
+      resourceNames:
+      - istio-galley
+      resources:
+      - deployments
+      verbs:
+      - get
+    {{< /text >}}
+
+    `istio-galley` needs `validatingwebhookconfigurations` write access to
+    create and update the `istio-galley` `validatingwebhookconfiguration`.
+
+## Creating configuration fails with `no such hosts` or `no endpoints available` errors
+
+Validation is fail-close. If the `istio-galley` pod is not ready,
+configuration cannot be created and updated.  In such cases you’ll see
+an error about `no such host` (Kubernetes 1.9) or `no endpoints
+available` (>=1.10).
+
+Verify the istio-galley pods(s) are running and endpoints are ready.
```

Comment:
Backticks around istio-galley

Could you search and make sure all uses of istio-galley are in backticks, and naked uses of Galley are capitalized?